benchmark/micro/json/RealNest/1_unnest_and_sum_one_json_struct.benchmark

```diff
@@ -0,0 +1,17 @@
+# name: benchmark/micro/json/RealNest/1_unnest_and_sum_one_json_struct.benchmark
+# description: Create a view reading a JSON file, unnesting one of structures and summing its fields values
```

Comment:
In addition to aggregating (`SUM`) the fields after unnesting, we can add a run computing the aggregate directly on the `STRUCT`.
```sql
SELECT i, SUM(s.a), SUM(s.b) FROM my_tbl GROUP BY i;
```

And a minor nit: I was confused because we do not create an SQL `VIEW` here; it is just a table.

Reply:
Thank you! I was too focused on unnesting everything))) With the applying aggregation directly on the `struct` it looks much cleaner and runs faster, but do we need a benchmark of this kind?

Reply:
Yes, we should keep some benchmarks for `UNNEST`. But others should not need any unnesting. :)

Reply:
Should we keep both variants: aggregating after unnesting and direct aggregating?

Reply:
Yes, I'd say so.